.github/workflows/e2e-tests.yml

```
@@ -93,7 +85,7 @@ jobs:
       with:
         tool: wasm-pack
     - name: Cache Wasm (because rust diff)
-      if: needs.check-rust-changes.outputs.rust-changed == 'true'
+      if: github.event_name != 'schedule' && steps.filter.outputs.rust == 'true'
```

Looks like we always build wasm below when scheduled. So shouldn't this be...?
```suggestion
      if: github.event_name == 'schedule' || steps.filter.outputs.rust == 'true'
```